### ISSUE TITLE
Use appstream translations

### ISSUE THREAD

**Issue by milotype** [flathub-infra/website#1058]
Use translated strings for App titles and descriptions in localized web pages, just as they are used on the GNOME web page (using `appdata.xml.in` or `desktop.in` files, or such data from the `.po` or other translation files).

**Examples for Croatian:**

Currently on the _beta_ **Flathub** page, app titles and descriptions do not use translated strings:
- https://beta.flathub.org/hr/apps/category/System
- https://beta.flathub.org/hr/apps/org.gnome.baobab

On the **GNOME** page, app titles and descriptions use translated strings:
- https://apps.gnome.org/hr/
- https://apps.gnome.org/hr/app/org.gnome.baobab/

Sorry, I'm not a programmer, so I don't know how the **GNOME** page is handling such data.

**Comment by razzeee** [comment]
See https://github.com/flathub/website/pull/88 

tldr; it's unclear, if this will improve the experiance, at least not without adding more quality gates to the flathub submission flow

**Comment by AsciiWolf** [comment]
It will definitely improve the user experience for users who do not speak English. It does not make much sense to keep the AppStream metadata English-only when the Flathub.org page itself is translated, at least in my opinion.

**Comment by wjt** [comment]
I think what @razzeee is alluding to is: many apps on Flathub – perhaps even the majority? – don't provide translations for their AppSteam metadata. So if the Flathub website used the translated information where present, it would in practice be a mixture of English and local language across different apps. And maybe even within the same app – for example I believe that the Gnome translation infrastructure deliberately does not translate app release notes, for reasons I can't recall.

English is my first language so I don't have a good sense of whether partial translations are better than nothing. Personally I think it makes sense to use the translations if available (which also would provide a bigger incentive to translate this data) but my opinion should be weighted very low!

**Comment by razzeee** [comment]
I still think any translation is better then non, for people that don't speak english. Stuff not being translated at all/wrong/awkward is moot at best.

**Comment by ikolomiko** [comment]
@razzeee @barthalion Any updates on this? Given that pretty much all application store frontends use the translations provided by the already **existing** `.metainfo.xml` files and the fact that a possible fix (#88) for this problem is being hold for 2 years, the current situation with Flathub is simply ridiculous.
As others pointed out in this conversation, the user experience for non-english speakers is terrible, to say the least. Even the slightest improvement on l10n would be greatly appreciated by millions of Linux users worldwide!

**Comment by secext2022** [comment]
Please add this feature, this is very important for non-English users.

**Comment by gaboolic** [comment]
Please add this feature, this is very important for non-English users.
The function of flathub.org have to switch the language (https://flathub.org/languages).
However, the app detail page only reads the introduction of the app in English, is this a bit schizophrenic?

**Comment by razzeee** [comment]
We're shipping this now.

Nextup is using translated fields in searches, but this was about the appstream translations, so closing it.
